charts/actions-runner-controller/templates/githubwebhook.secrets.yaml

```diff
@@ -12,5 +12,20 @@ data:
 {{- if .Values.githubWebhookServer.secret.github_webhook_secret_token }}
   github_webhook_secret_token: {{ .Values.githubWebhookServer.secret.github_webhook_secret_token | toString | b64enc }}
 {{- end }}
+{{- if .Values.githubWebhookServer.secret.github_app_id }}
+  github_app_id: {{ .Values.githubWebhookServer.secret.github_app_id | toString | b64enc }}
+{{- end }}
+{{- if .Values.githubWebhookServer.secret.github_app_installation_id }}
+  github_app_installation_id: {{ .Values.githubWebhookServer.secret.github_app_installation_id | toString | b64enc }}
+{{- end }}
+{{- if .Values.githubWebhookServer.secret.github_app_private_key }}
+  github_app_private_key: {{ .Values.githubWebhookServer.secret.github_app_private_key | toString | b64enc }}
+{{- end }}
+{{- if .Values.githubWebhookServer.secret.github_token }}
+  github_token: {{ .Values.githubWebhookServer.secret.github_token | toString | b64enc }}
+{{- end }}
+{{- if .Values.githubWebhookServer.secret.github_basicauth_password }}
+  github_basicauth_password: {{ .Values.githubWebhookServer.secret.github_basicauth_password | toString | b64enc }}
+{{- end }}
```

Comment:
Although it's harmless, strictly speaking, we might not need this at the moment, as we seem to refer to the auth secret for this in the webhook server deployment spec anyway:

https://github.com/actions-runner-controller/actions-runner-controller/blob/bfc5ea4727f3dfb1a8cc590306f04f3d32133d4b/charts/actions-runner-controller/templates/githubwebhook.deployment.yaml#L106

Reply:
Oh. Maybe I should delete it to avoid confusion